packages/core/src/pubsub/pubsub-keepalive.ts

```diff
@@ -60,12 +66,18 @@ export class PubsubKeepalive
    */
   publishPubsubKeepaliveIfNeeded(): void {
     const now = Date.now()
-    if (now - this.lastPublishedMessageDate < this.maxPubsubPublishInterval / 2) {
-      // We've published a message recently enough, no need to publish another
+    const needToPublishKeepaliveOnceADay =
```

Comment:
please pull the interval into a constant `MIN_KEEPALIVE_INTERVAL`

Reply:
pulled it out, but named it `ONE_DAY_MS` for clearer naming